resources/views/components/sidemenus/content-moderator.blade.php

```diff
@@ -46,5 +95,18 @@
             :route="'moderator.schedule.index'"
             :icon="'M6.75 3v2.25M17.25 3v2.25M3 18.75V7.5a2.25 2.25 0 012.25-2.25h13.5A2.25 2.25 0 0121 7.5v11.25m-18 0A2.25 2.25 0 005.25 21h13.5A2.25 2.25 0 0021 18.75m-18 0v-7.5A2.25 2.25 0 015.25 9h13.5A2.25 2.25 0 0121 11.25v7.5'"
             :roleColour="'crew'"></x-sidebar-link-content-mod>
+        <x-sidebar-link-content-mod
+            :label="'Editions'"
+            :route="'moderator.editions.index'"
+            :icon="'M6.75 3v2.25M17.25 3v2.25M3 18.75V7.5a2.25 2.25 0 012.25-2.25h13.5A2.25 2.25 0 0121 7.5v11.25m-18 0A2.25 2.25 0 005.25 21h13.5A2.25 2.25 0 0021 18.75m-18 0v-7.5A2.25 2.25 0 015.25 9h13.5A2.25 2.25 0 0121 11.25v7.5'"
+            :roleColour="'crew'"></x-sidebar-link-content-mod>
+        @if (Edition::current())
+            <x-sidebar-link-content-mod
+                :label="'Events'"
+                :route="'moderator.events.index'"
+                :param="Edition::current()"
+                :icon="'M6.75 3v2.25M17.25 3v2.25M3 18.75V7.5a2.25 2.25 0 012.25-2.25h13.5A2.25 2.25 0 0121 7.5v11.25m-18 0A2.25 2.25 0 005.25 21h13.5A2.25 2.25 0 0021 18.75m-18 0v-7.5A2.25 2.25 0 015.25 9h13.5A2.25 2.25 0 0121 11.25v7.5'"
+                :roleColour="'crew'"></x-sidebar-link-content-mod>
+        @endif
```

Comment:
Can we do other icons instead of the same as the `Schedule Management`. Also I get the logic if there's a current edition to have the events more at a shortcut, but I think it might get confusing. Rather leave just the `Editions` link.

Reply:
Yeah as I said in the description I just copied the icons, since Idk where we got the icons from and I am not a design person to make this life-changing choice :)